File: c/soma_vetor.c
#include <stdio.h>


int main()
{
    int n;
    double x, soma, media, counter;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    double vet[n];

    for(int i = 0; i < n; i++) {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    printf("\nVALORES = ");
    for(int i = 0; i < n; i++) {
        printf("%.2lf ", vet[i]);
    }

    soma = 0;
    printf("\nSOMA = ");
    for(int i = 0; i < n; i++) {
        soma = soma + vet[i];
        counter++;
    }
    printf("%.2lf", soma);
    media =  soma / counter;
    printf("\nMEDIA = %.2lf", media);






    return 0;
}

File: csharp/soma_vetor.cs
using System;
using System.Globalization;

namespace soma_vetor
{
    internal class Program
    {
        static void Main(string[] args)
        {

            CultureInfo CI = CultureInfo.InvariantCulture;

            Console.Write("Quantos numeros voce vai digitar? ");
            int n = int.Parse(Console.ReadLine());

            double[] vet = new double[n];

            for (int i = 0; i < n; i++)
            {
                Console.Write("Digite um numero: ");
                vet[i] = double.Parse(Console.ReadLine(), CI);
            }

            Console.WriteLine();
            Console.Write("VALORES = ");
            for (int i = 0; i < n; i++)
            {
                Console.Write(vet[i].ToString("F1", CI) + " ");
            }

            double soma = 0;

            for (int i = 0;i < n; i++)
            {
                soma = soma + vet[i];            
            }
            Console.WriteLine();
            Console.WriteLine("SOMA = " + soma.ToString("F2", CI));

            double media = soma / n;
            Console.WriteLine("MEDIA = " +  media.ToString("F2", CI));
        }
    }
}
